packages/cactus-cmd-api-server/src/test/typescript/integration/jwt-endpoint-authz-scope-enforcement.test.ts

```diff
@@ -142,22 +139,12 @@ test(testCase, async (t: Test) => {
           Authorization: `Bearer ${token}`,
         },
       });
-      t.fail("deploy contract response status === 403 FAIL");
-    } catch (out) {
-      t.ok(out, "error thrown for forbidden endpoint truthy OK");
-      t.ok(out.response, "deploy contract response truthy OK");
-      t.equal(
-        out.response.status,
-        StatusCodes.FORBIDDEN,
-        "deploy contract response status === 403 OK",
-      );
-      t.notok(out.response.data.data, "out.response.data.data falsy OK");
-      t.notok(out.response.data.success, "out.response.data.success falsy OK");
+    } catch (out: any) {
```

Comment:
@Leeyoungone This can be made to work with `unkownn` as the type of the exception instead of `any`. Do you remember that pattern I showed earlier? Not sure when it was TBH.